server/odc-migrate/src/main/resources/migrate/common/V_4_2_0_42__alter_partition_plan.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `table_partition_plan` MODIFY COLUMN `flow_instance_id` bigint(20) COMMENT '任务 ID' NULL;
```

Comment:
if a column should not be null, use `NOT NULL`

Reply:
done